dataTiger.py

```diff
@@ -169,6 +169,66 @@ def downloadStandardCurveReport(self):
         else:
             self.statusUpdateText.append("Unable to download missing analyzer tests. Invalid database.\n\n")
 
+    def sendToUVU(self):
```

Comment:
This part obviously isn't ready! Mary showed me a file of all the data we need to send to UVU and it kind of seemed like it was just everything, is that right?